src/lib/__tests__/Runner.test.ts

```diff
@@ -439,6 +443,23 @@ describe('Runner', () => {
       );
     });
 
+    test('should fail land request if timeout period configured via config is breached', async () => {
+      mockConfig.prSettings.landBuildTimeoutTime = 1000 * 60 * 60;
+      //running state is beyond the timeout period of 2 hours
+      const mockLandRequestStatus = getLandRequestStatus(new Date('2022-12-13T03:42:48.071Z'));
```

Comment:
I'm curious why this specific date?

Reply:
The code looks for `current date - this date` to calculated the time elapsed. We haven't mocked the current date. So, any timestamp in past should be sufficient to test the scenario